src/main/java/org/cloudfoundry/promregator/config/OAuth2XSUAAAuthenticationConfiguration.java

```diff
@@ -1,45 +1,114 @@
 package org.cloudfoundry.promregator.config;
 
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.http.client.utils.URIBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class OAuth2XSUAAAuthenticationConfiguration {
-	private String tokenServiceURL;
-	
+
+	public final static String deprecatedTokenServciceURLProperty = "tokenServiceURL";
+	public final static String useInsteadXsuassServiceURLProperty = "xsuaaServiceURL";
```

Comment:
Constants should all be uppercase using underscores as separators between words.

Reply:
```suggestion
	public final static String deprecatedTokenServciceURLProperty = "tokenServiceURL";
	public final static String useInsteadXsuaaServiceURLProperty = "xsuaaServiceURL";
```